app/services/age_calculator.rb

```diff
@@ -0,0 +1,12 @@
+class AgeCalculator
```

Comment:
This is great, I think is more clear and as you mention might be differences depending on < 18 or < 16 so having this (even admittedly simple logic) in a separate class is cleaner.

Perhaps only suggestion, not important really, the class could be initialised with a person (in this case applicant) record directly instead of the whole crime application?
That way is easier to calculate dates for other `Person` subclasses like perhaps, if needed in the future, partners. As we don't couple the class to `crime_application.applicant`

Reply:
👍 I didn't think that through, thinking I didn't want to initialise with `applicant` - but yes, 100% thank you